setup.py

```diff
@@ -84,6 +84,7 @@ def run(self):
         "dash-core-components",
         "dash-html-components",
         "dash",
+        "dataclasses",  # needed for python < 3.7
```

Comment:
Can we make python 3.7+ a hard dependency?

Reply:
Good question, does not matter to me. I guess that should be answered by our intended audience, same as things like officially supported Windows, etc.